pkg/client/simple/vfsclientset/instancegroup.go

```diff
@@ -36,6 +37,27 @@ type InstanceGroupVFS struct {
 	clusterName string
 }
 
+type InstanceGroupMirror interface {
+	WriteMirror(ig *kops.InstanceGroup) error
+}
+
+var _ InstanceGroupMirror = &InstanceGroupVFS{}
+
+func NewInstanceGroupMirror(clusterName string, configBase vfs.Path) InstanceGroupMirror {
+	kind := "InstanceGroup"
+
+	r := &InstanceGroupVFS{
+		clusterName: clusterName,
+	}
+	r.init(kind, configBase.Join("instancegroup"), StoreVersion)
+	defaultReadVersion := v1alpha1.SchemeGroupVersion.WithKind(kind)
+	r.defaultReadVersion = &defaultReadVersion
+	r.validate = func(o runtime.Object) error {
+		return validation.ValidateInstanceGroup(o.(*kops.InstanceGroup))
```

Comment:
Should we test if `o.(*kops.InstanceGroup)` is a `*kops.InstanceGroup`?

Reply:
No, it would be a bug, and panic-ing is appropriate here